### PR TITLE
feat: add import and export by new props and define tooltips text in …

### PR DESCRIPTION
**Pull request by Guismus**
…translation files

Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

**Comment by Guismus** [comment]
Cypress test not fixed, WIP

File: public/locales/fr/translation.json [hunk 1]
```diff
@@ -555,7 +555,12 @@
         "fileImportError": "Échec du chargement du fichier.",
         "maxErrorsCount": "(seulement les {{maxCount}} premiers résultats)",
         "errorsCount_one": "{{count}} erreur:",
-        "errorsCount_other": "{{count}} erreurs:"
+        "errorsCount_other": "{{count}} erreurs:",
+        "placeholdertitle": "Importer votre premier fichier",
+        "placeholderbody": "Une fois que vous ayez importé votre premier fichier csv ou xls, vos données vont être affiché dans un tableau intéractif.",
```
Comment (esasova) [inline public/locales/fr/translation.json:560]: typo: données vont être affich**ées** and int**E**ractif (without **é**)

**Review by esasova — APPROVED** [review]
some remarks but LGTM :+1:

File: public/locales/fr/translation.json (hunk 1 above)
Comment (Guismus) [reply public/locales/fr/translation.json:560]: DONE

**Review by csm-thu — CHANGES_REQUESTED** [review]

File: src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js [hunk 2]
```diff
@@ -498,7 +471,8 @@ export const GenericTable = ({
         columns={columns}
         rows={parameter.agGridRows || []}
         agTheme={context.isDarkTheme ? gridDark.agTheme : gridLight.agTheme}
-        extraToolbarActions={extraToolbarActions}
+        onImport={importFile}
+        onExport={openExportDialog}
```
[the remark this answers is not in the document]
Comment (Guismus) [reply src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js:492]: DONE

File: public/locales/fr/translation.json [hunk 3]
```diff
@@ -555,7 +555,12 @@
         "fileImportError": "Échec du chargement du fichier.",
         "maxErrorsCount": "(seulement les {{maxCount}} premiers résultats)",
         "errorsCount_one": "{{count}} erreur:",
-        "errorsCount_other": "{{count}} erreurs:"
+        "errorsCount_other": "{{count}} erreurs:",
+        "placeholdertitle": "Importer votre premier fichier",
+        "placeholderbody": "Une fois que vous ayez importé votre premier fichier csv ou xls, vos données vont être affichées dans un tableau interactif.",
```
[the remark this answers is not in the document]
Comment (Guismus) [reply public/locales/fr/translation.json:560]: DONE

File: public/locales/en/translation.json [hunk 4]
```diff
@@ -554,7 +554,12 @@
         "fileImportError": "File load failed.",
         "maxErrorsCount": "(only the top first {{maxCount}} results)",
         "errorsCount_one": "{{count}} error occurred:",
-        "errorsCount_other": "{{count}} errors occurred:"
+        "errorsCount_other": "{{count}} errors occurred:",
+        "placeholdertitle": "Import your first data",
+        "placeholderbody": "After importing a valid csv or xls file, your data will be displayed in an interactive table.",
```
[the remark this answers is not in the document]
Comment (Guismus) [reply public/locales/en/translation.json:559]: DONE

File: src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js [hunk 5]
```diff
@@ -63,6 +62,11 @@ export const GenericTable = ({
     loading: t('genericcomponent.table.labels.loading', 'Loading...'),
     clearErrors: t('genericcomponent.table.button.clearErrors', 'Clear'),
     errorsPanelMainError: t('genericcomponent.table.labels.fileImportError', 'File load failed.'),
+    placeholderTitle: t('genericcomponent.table.labels.placeholdertitle'),
+    placeholderBody: t('genericcomponent.table.labels.placeholderbody'),
+    import: t('genericcomponent.table.labels.import'),
+    export: t('genericcomponent.table.labels.export'),
+    fullscreen: t('genericcomponent.table.labels.fullscreen'),
```
[the remark this answers is not in the document]
Comment (Guismus) [reply src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js:69]: DONE

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**